execjs/_runtimes.py

```diff
@@ -10,6 +10,29 @@
 import execjs._pyv8runtime as pyv8runtime
 
 
+class RuntimeNames(object):
```

Comment:
This class is not Pythonic. This class should be a module rather than class because it never instantiate.